### PR TITLE
[IMP] mail: fetch init mail failures in another RPC

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2742946
Task-2746116

### CODE REVIEW

File: addons/mail/static/tests/chatter_tests.js [hunk 1]
```diff
@@ -174,13 +175,14 @@ QUnit.test('list activity widget with activities', async function (assert) {
     assert.verifySteps([
         '/web/dataset/search_read',
         '/mail/init_messaging',
+        '/mail/init_messaging_mail_failures',
```
Comment (seb-odoo) [inline addons/mail/static/tests/chatter_tests.js:178]: I would rather if we didn't `assert.step` every route instead, this is completely irrelevant to this test.

File: addons/mail/static/tests/chatter_tests.js [hunk 2]
```diff
@@ -212,13 +214,14 @@ QUnit.test('list activity widget with exception', async function (assert) {
...
```
Comment (seb-odoo) [inline addons/mail/static/tests/chatter_tests.js:217]: I would rather if we didn't `assert.step` every route instead, this is completely irrelevant to this test.

**Review by seb-odoo — COMMENTED** [review]
comments apply for v15 (but posting on the master pr...)

**Comment by alexkuhn** [comment]
Changes done (also applied on 15.0 branch)

**Pull request closed without merging**